chalicelib/checks/release_updates_checks.py

```diff
@@ -498,7 +499,9 @@ def data_release_updates(connection, **kwargs):
         return check
     tag_filter = kwargs['tag_filter']
     project_filter = kwargs.get('project_filter')
-    used_res_strings = ' Compared results from %s (start) to %s (end). UUIDs are %s (start) and %s (end). Used filter on replicate sets tag is %s and the filter on award.project is %s (None means no filter in both cases).' % (start_date_str, end_date_str, start_data_result.get('uuid', 'None'), end_data_result['uuid'], tag_filter, project_filter)
+    used_res_strings = ' Compared results from %s (start) to %s (end). UUIDs are %s (start) and %s (end). Used filter on replicate sets tag is %s and the filter on award.project is %s (None means no filter in both cases).' % (
```

Comment:
With a lot of interpolations like this, using a dictionary on the RHS of the `%` and using `%(key)s` in the text might be clearer, but I guess this is OK.